tutorials/02_intermediate/color_cycle.py

```diff
@@ -0,0 +1,124 @@
+"""
+===================
+Styling with cycler
+===================
+
+Demo of custom property-cycle settings to control colors and other style
+properties for multi-line plots.
+
+.. note:: More complete documentation of the `cycler` API can be found
+          `here <http://matplotlib.org/cycler/>`_.
+
+This example demonstrates two different APIs:
+
+    1. Setting the default rc parameter specifying the property cycle.
+       This affects all subsequent axes (but not axes already created).
+    2. Setting the property cycle for a single pair of axes.
+
+"""
+from cycler import cycler
+import numpy as np
+import matplotlib.pyplot as plt
+
+###############################################################################
+# First we'll generate some sample data, in this case, four offset sine
+# curves.
+x = np.linspace(0, 2 * np.pi, 50)
+offsets = np.linspace(0, 2*np.pi, 4, endpoint=False)
+yy = np.transpose([np.sin(x + phi) for phi in offsets])
+
+###############################################################################
+# Now `yy` has shape
+print(yy.shape)
+
+###############################################################################
+# So `yy[:,i]` will give you the `i`-th offset sine curve. Let's set the
+# default prop_cycle using `plt.rc`. We'll combine a color cycler and a
```

Comment:
here could you try linking to rc with an rST `:ref:`? I think that'll make it link nicely in the docs